okta/resource_app_oauth.go

```diff
@@ -235,6 +237,12 @@ func resourceAppOAuth() *schema.Resource {
 				Default:     true,
 				Description: "Do not display application icon to users",
 			},
+			"custom_profile_attributes": &schema.Schema{
```

Comment:
Nit pick, could we rename this to `profile`? We used this name for users primarily because it only accepts "custom" user schema property values. This one is the entire profile.

Reply:
Updated